WORKLIST.txt

```diff
@@ -48,6 +49,10 @@ PLAYER:
 - equipped items are used to make attacks and to defend against attacks (raise AC, raise to-hit and damage stats, &c)
 - equippable items restricted to certain subset of all items ITEM_FLAG_EQUIPMENT, instead of hardcoded in CPlayer::Wield.
 - implement paginated inventory
+- how to remove cursed?
```

Comment:
cursed items can be removed with the spell of Remove Curse (not implemented), that can either be cast by the player (if of a class that can do such things: Priest / Mage maybe, but not Warrior), or read from a Scroll of Remove Curse (a low-level scroll that is usually available either at the Church or the Magic Shop in town, or can be found in the dungeon). Scrolls show up "un-identified", so the first time you see it in-dungeon, it will look like "Scroll of flaz-bat-mog" and then you either have to Identify it (by casting, reading a scroll, or using a Staff or Gem) or read it blindly, and hope it's not a Scroll of Curse Weapon or a Scroll of Summon Monster or a Scroll of Create Traps (which are invisible until discovered). "Cursed" is part of the gameplay, that the *player* should overcome, not the dev.